### PR TITLE
Chore(ns-mode): Add manifest for namespace mode of litmus portal with some fixes on rbac

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: udit <udit@chaosnative.com>

## Proposed changes

- Add `docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml` litmus portal manifest for namespace operator.
- Update permission to run chaos experiments with operator ns service account permission.
- Fix: Replace cluster role with role in the namespaced mode of execution. 

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by ishangupta-ds — DISMISSED** [review]
@uditgaurav can we remove the namespaced template file then, from the litmus-portal disrectory? Cc: @rajdas98

**Comment by uditgaurav** [comment]
@ishangupta-ds we can keep it to run with the `ci` tag and `docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml` will be released version.

**Review by ishangupta-ds — APPROVED** [review]
LGTM

**Review by gdsoumya — APPROVED** [review]
lgtm

File: docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: litmus-server
+  labels:
+    name: litmus-server
+rules:
+  - apiGroups:
+      - "*"
```
Comment (ksatchit) [inline docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml:72]: Why is this `*/*/*`

**Review by rajdas98 — CHANGES_REQUESTED** [review]
@uditgaurav Can you remove these two from the directory?
- https://github.com/litmuschaos/litmus/blob/master/litmus-portal/namespaced-K8s-template.yml
- https://github.com/litmuschaos/litmus/blob/master/litmus-portal/cluster-k8s-manifest.yml

and use the GitHub pages manifest in the portal readme. Having two similar manifest will become a complexity to manage.

File: docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml (hunk 1 above)
Comment (uditgaurav) [reply docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml:72]: @ksatchit  this require permission to create role and rolebindings and it should have all the permission that is inside the rbac we are creating so eventually it becomes very large and complex so we gave all and checking the upper limit from other rbac. We do same thing in cluster mode as well.

**Review by rajdas98 — APPROVED** [review]

**Comment by rajdas98** [comment]
> @uditgaurav Can you remove these two from the directory?
> 
> * https://github.com/litmuschaos/litmus/blob/master/litmus-portal/namespaced-K8s-template.yml
> * https://github.com/litmuschaos/litmus/blob/master/litmus-portal/cluster-k8s-manifest.yml
> 
> and use the GitHub pages manifest in the portal readme. Having two similar manifests will become a complexity to manage.

we can do it later

**Pull request merged**